data/README.rst

```diff
@@ -0,0 +1,107 @@
+Software Directory README
+#########################
+
+As decided by the XSF Board, the Software Directory will need yearly renewal for software to not disappear from the listing on the website. To help software authors with this, scripts are provided which help you with creating a proper pull request.
```

Comment:
I would remove "As decided by the XSF Board." It does not really matter who decided this for the README.